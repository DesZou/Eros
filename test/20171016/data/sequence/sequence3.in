50 11601769 17
327779815 929296701 705093054 365253017 721758804 114940757 627245027 723397734 882102110 934017149 395875358 384286636 167915473 248698973 30844947 73726351 599104097 723558669 90141000 193672643 131779330 961077720 845073055 336218504 677974144 244650894 805405665 329737919 451070515 956478366 144878248 49409018 82095541 97870578 86454819 833788861 40182353 887002948 123699840 926453276 377040886 394698724 296446489 559962763 348377736 198014247 349721128 407881158 732821330 507867434 
50
39 47
38 50
8 46
19 34
15 23
49 49
14 24
36 41
28 37
43 43
22 33
20 42
31 50
5 22
17 28
7 31
32 43
29 47
9 10
22 32
50 50
5 27
35 49
7 11
12 25
30 34
13 37
50 50
16 17
32 39
11 17
27 39
6 10
22 46
14 29
46 47
16 27
3 33
32 43
29 32
36 45
5 42
1 14
50 50
18 31
40 50
40 45
50 50
36 42
20 22
