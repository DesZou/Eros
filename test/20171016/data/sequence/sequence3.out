0
0
276
0
0
0
0
0
0
0
0
28
10
3
0
45
0
6
0
0
0
28
0
0
0
0
45
0
0
0
0
0
0
45
0
0
0
120
0
0
0
253
0
0
0
0
0
0
0
0
